components/core/src/clp_s/TimestampDictionaryReader.cpp

```diff
@@ -43,12 +45,13 @@ void TimestampDictionaryReader::read_new_entries(bool local) {
     }
 
     for (int i = 0; i < range_index_size; ++i) {
-        std::string col;
+        std::string column_name;
+        std::unordered_set<int32_t> column_ids;
```

Comment:
Do we want to add a getter for the column_ids, or leave that for when the PR that will use it?

Reply:
I plan to add it in the next PR. If we assume that the first range index is from the authorized key, we can just store the first column_ids in TimestampReader